src/styles/globals.css

```
@@ -193,3 +193,18 @@ html{
 .mouse-cursor-gradient-tracking > * {
   transition: opacity 0.2s ease-in-out;
 }
+
+.jellyfish-item {
+  animation: diagonal-move-anim 2.5s cubic-bezier(0.42, 0, 0.58, 1) infinite alternate;
+  position: absolute;
+}
+
+@keyframes diagonal-move-anim {
+  0% {
+    transform: translate(150px, 0px);
+  }
+
+  100% {
+    transform: translate(50px, 50px);
```

just something subtle
```suggestion
    transform: translate(130px, 20px);
```